components/root-actions/AccountType.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { gql, useMutation } from '@apollo/client';
+import { useIntl } from 'react-intl';
+
+import { i18nGraphqlException } from '../../lib/errors';
+import { API_V2_CONTEXT } from '../../lib/graphql/helpers';
+
+import CollectivePickerAsync from '../CollectivePickerAsync';
+import ConfirmationModal from '../ConfirmationModal';
+import StyledButton from '../StyledButton';
+import StyledInputField from '../StyledInputField';
+import { P } from '../Text';
+import { TOAST_TYPE, useToasts } from '../ToastProvider';
+
+export const editAccountTypeMutation = gql`
+  mutation EditAccountType($account: AccountReferenceInput!) {
+    editAccountType(account: $account) {
+      id
+      slug
```

Comment:
Might be good to get the new type to refresh it in local cache

```suggestion
      slug
      type
```